src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/paymentsDetails.js

```diff
@@ -0,0 +1,31 @@
+const Logger = require('dw/system/Logger');
+const URLUtils = require('dw/web/URLUtils');
+const AdyenHelper = require('*/cartridge/scripts/util/adyenHelper');
+const adyenCheckout = require('*/cartridge/scripts/adyenCheckout');
+
+/*
+ * Makes a payment details call to Adyen to confirm redirectResults and returns the resultCode
+ */
+function paymentsDetails(req, res, next) {
```

Comment:
Maybe in the comments here we should explain why this is needed. Explain that there is no order but we still need to make a paymentsDetails call

Reply:
I'll elaborate on what this is used for. Note that we might use this in the future for our regular 3DS payments flow as well